ui/components/app/name/name-details/index.scss

```diff
@@ -7,4 +7,9 @@
   &__address .mm-text-field {
     padding-right: 5px;
```

Comment:
Can we not use the provided style utility props for this `<TextField paddingRight={1}` `1` = `4px`

Reply:
The `<TextField>` rendered by the `<FormTextField>` so we don't have direct access to it in the `NameDetails` render method.